tests/RequestTest.php

```diff
@@ -196,4 +196,30 @@ public function testGetAccept()
         $this->assertEquals('accept', $this->request->getAccept('default'));
     }
 
+    public function testContentRange()
+    {
+        $_SERVER['HTTP_CONTENT_RANGE'] = "bytes 0-499/2000";
+        
+        $this->assertEquals('bytes', $this->request->getContentRangeUnit());
+        $this->assertEquals(0, $this->request->getContentRangeStart());
+        $this->assertEquals(499, $this->request->getContentRangeEnd());
+        $this->assertEquals(2000, $this->request->getContentRangeSize());
+
+        $_SERVER['HTTP_CONTENT_RANGE'] = " 0-499/2000";
+        
+        $this->request = new Request();
+        $this->assertEquals(null, $this->request->getContentRangeUnit());
+        $this->assertEquals(null, $this->request->getContentRangeStart());
+        $this->assertEquals(null, $this->request->getContentRangeEnd());
+        $this->assertEquals(null, $this->request->getContentRangeSize());
+
+        $_SERVER['HTTP_CONTENT_RANGE'] = "bytes 0-499/";
```

Comment:
Does it make sense to test headers that doesn't resemble the standarts?

- `bytes 0--499/2000`
- `bytes 0-499/20/00`
- `bytes 0-499test/2000`
- `bytes 0-49.9/200.0`
- `bytes 0-49,9/200,0`

Reply:
May be I should thanks for the ideas